doc/book/src/dev/platform-tier-policy.md

```diff
@@ -0,0 +1,136 @@
+stronger requirement.
+
+While these criteria attempt to document the policy, that policy still involves human
+judgment. Platforms must fulfill the spirit of the requirements as well, as determined by
+the judgment of the approving reviewers. Reviewers and team members evaluating platforms
+and platform-specific patches should always use their own best judgment regarding the
+quality of work, and the suitability of a platform for the Zcash project. Neither this
+policy nor any decisions made regarding platforms shall create any binding agreement or
+estoppel by any party.
+
+For a list of all supported platforms and their corresponding tiers ("tier 3", "tier 2",
+or "tier 1"), see [Platform Support](../user/platform-support.md).
+
+The availability or tier of a platform in Zcash releases is not a hard stability guarantee
+about the future availability or tier of that platform. Higher-level platform tiers are an
+increasing commitment to the support of a platform, and we will take that commitment and
+potential disruptions into account when evaluating the potential demotion or removal of a
+platform that has been part of a stable release. The promotion or demotion of a platform
+will not generally affect existing stable releases, only current development and future
+releases.
+
+In this policy, the words "must" and "must not" specify absolute requirements that a
+platform must meet to qualify for a tier. The words "should" and "should not" specify
+requirements that apply in almost all cases, but for which the approving teams may grant
+an exception for good reason. The word "may" indicates something entirely optional, and
+does not indicate guidance or recommendations. This language is based on
+[IETF RFC 2119](https://tools.ietf.org/html/rfc2119).
+
+## Tier 3 platform policy
+
+At this tier, ECC provides no official support for a platform, so we place minimal
+requirements on the introduction of platforms.
+
+A proposed new tier 3 platform must be reviewed and approved by a member of the ECC core
+team based on these requirements.
+
+- The platform must provide documentation for the Zcash community explaining how to build
+  for the platform, using cross-compilation if possible. If the platform supports running
+  binaries, or running tests (even if they do not pass), the documentation must explain
+  how to run such binaries or tests for the platform, using emulation if possible or
+  dedicated hardware if necessary.
+- Tier 3 platforms must not impose burden on the authors of pull requests, or other
+  developers in the community, to maintain the platform. In particular, do not post
+  comments (automated or manual) on a PR that derail or suggest a block on the PR based on
+  a tier 3 platform. Do not send automated messages or notifications (via any medium,
+  including via @) to a PR author or others involved with a PR regarding a tier 3
+  platform, unless they have opted into such messages.
```

Comment:
I don't see why reviewers should avoid commenting on portability to tier 3 platforms; that seems like a reasonable kind of review comment to me. It's not the same as blocking on support for a tier 3 platform.

Reply:
Commenting at all on a PR notifies the PR author, though. And if you consider a PR author to have opted into notifications, then the "don't notify the PR author" part of this becomes literally a no-op. I have no objection to saying that the PR shouldn't be blocked, that's not what my comment was about.

Reply:
It covers everything other than PR authors in their own PRs though (e.g. other issues, including the one that a PR might be addressing), so without any suggested alternative wording, I'm content to leave it as more restrictive by default, and then handle any issues / relaxations as they arise.